### DISCUSSION TITLE
Performance benefits of creating your site with WordPress in 2023

### DISCUSSION THREAD

**Discussion by juanmaguitar** [WordPress/developer-blog-content#99] [Topic Ideas]
About Performance with Wordpress 6.2:
- [Measuring website performance using WebPageTest (with WordPress 6.2)](https://felix-arntz.me/blog/measuring-website-performance-using-webpagetest-with-wordpress-6-2/)
- [WordPress 6.2 Performance improvements for all themes](https://make.wordpress.org/core/2023/04/05/wordpress-6-2-performance-improvements-for-all-themes/)
- [Evolution of WordPress TTFB: 5.6 to 6.2](https://oandre.gal/evolution-of-wordpress-ttfb-5-6-to-6-2/)

Performance is one of the concerns of developers so I think there's enough material to write about the performance benefits of using WordPress 6.2 to build a site.

Some other talking points for the article:
- Performance of WordPress Sites compared to Static Sites (Reference: [Turn Your WordPress Site Into a Static Site With Aggressive Nginx Page Caching](https://spinupwp.com/wordpress-static-site-nginx-page-caching/))

**Comment by justintadlock** [comment]
This seems to almost be a repeat of the [earlier ticket around performance](https://github.com/WordPress/developer-blog-content/issues/14). The previous ticket was specific to block themes, and this is more general.

Extenders who read this blog are most likely already going to be using WordPress 6.2, so I feel like the general pitch is too vague. Is the expected audience non-WP developers who we want to start building with WP?

If it's a WP-specific audience, I would ask: _Can I educate extenders on specific tools, techniques, etc. that they can use to build more performant sites?_

**Edit:** For example, the [Turn Your WordPress Site Into a Static Site With Aggressive Nginx Page Caching](https://spinupwp.com/wordpress-static-site-nginx-page-caching/) article is a great example of educating on a specific topic.

Perhaps @oandregal or @felixarntz would be interested in following up with a Dev Blog post??

**Comment by oandregal** [comment]
The earlier topic was a great read, thanks for preparing it.

> Can I educate extenders on specific tools, techniques, etc. that they can use to build more performant sites?

I'm unaware of other specific things related to how to build (block) themes, etc. that would have impact in performance.

The one thing that has immediate effect and is easy: set up a full page cache plugin. Felix described this setup a while ago https://felix-arntz.me/blog/rebuilding-my-website-using-a-block-theme/ It is somehow related to the topic of "static WordPress", though it involves installing and setting up a plugin, which I don't know whether it plays well with developer blog editorial direction (you may need to pick one plugin over the other).

I don't know that I have much other advice to share.

**Reply by justintadlock** [reply]
Generally, we try to avoid specifically highlighting plugins (unless it's a core plugin owned by the WordPress account). But, there may be situations where it's necessary (CC: @bph).

But, it might be worthwhile to explore some topics around caching.

**Comment by bph** [comment]
With the published post [Leveraging theme.json and per-block styles for more performant themes](https://developer.wordpress.org/news/2022/12/07/leveraging-theme-json-and-per-block-styles-for-more-performant-themes/) I close it as resolved. For general perfomance updates, the Performance team does excellent work, which can be [reviewed via the Make core blog.](https://make.wordpress.org/core/tag/core-performance/). The need for specific performance topics would probably be better discussed in their meeting, before submitting it for consideration for the Dev Blog meeting.
